notebooker/web/routes/index.py

```diff
@@ -17,7 +17,7 @@ def index(subfolder: Optional[str] = None):
     The index page which shows cards of each report which has at least one result in the database.
     """
     username = request.headers.get("X-Auth-Username")
-    all_reports = get_all_possible_templates()
+    all_reports = get_all_possible_templates(subfolder=subfolder)
```

Comment:
This just gives us the list of possible templates for executing a notebook, I don't think we need to filter it

Reply:
done